addons/mail/static/src/components/composer/composer.xml

```diff
@@ -128,7 +154,12 @@
                 <t t-if="hasFooter">
                     <div class="o_Composer_coreFooter" t-att-class="{ 'o-composer-is-compact': props.isCompact }">
                         <t t-if="props.hasThreadTyping">
-                            <ThreadTextualTypingStatus class="o_Composer_threadTextualTypingStatus" threadLocalId="composer.thread.localId"/>
+                            <t t-if="composer.thread">
+                                <ThreadTextualTypingStatus class="o_Composer_threadTextualTypingStatus" threadLocalId="composer.thread.localId"/>
+                            </t>
+                            <t t-if="composer.messageInEditing">
+                                <ThreadTextualTypingStatus class="o_Composer_threadTextualTypingStatus" threadLocalId="composer.messageInEditing.originThread.localId"/>
+                            </t>
```

Comment:
Same comment as with the 2 file uploaders, but here it's more duplicated info.

Haven't checked models, but I assume code's duplication but `composer.thread` and `composer.messageInEditing` is mutually exclusive? It's not obvious for me, especially when we designed `Composer` model as an arbitrary split from `Thread`.